src/main/java/server/response/ResponseTypes.java

```diff
@@ -50,4 +52,18 @@ public static Response internalError() {
                 .withStatus(VERSION + SP + INTERNAL_ERROR + CRLF)
                 .build();
     }
+
+    public static Response proxy(Request request, String s) {
+        String proxyPayload;
+        try {
+            proxyPayload = ProxyHandler.sendRequest(request);
+        } catch (IOException e) {
+            e.printStackTrace();
+            return ResponseTypes.internalError();
+        }
+        return new Response.Builder()
+                .withStatus(VERSION + SP + OK+ CRLF)
+                .withBody(proxyPayload)
+                .build();
```

Comment:
e.g. what tests that a proxy returns the payload on success and a 500 on failure?